iscservice/isc.c

```diff
@@ -136,12 +136,17 @@ static int isc_comp_launch(struct m0_isc_comp_req *comp_req, struct m0_fom *fom,
 	result = comp->ic_op(&comp_req->icr_args, &comp_req->icr_result,
 			     &comp_req->icr_comp_data, &rc);
 	M0_ASSERT(M0_IN(result, (M0_FSO_AGAIN, M0_FSO_WAIT)));
+	M0_LOG(M0_DEBUG, "ic_op(): rc=%d", rc);
 	/*
 	 * In case a re-invocation of a computation is necessary, phase
 	 * is not changed.
 	 */
 	if (rc == -EAGAIN)
 		return result;
+
+	if (rc != 0)
```

Comment:
Do you think it is helpful or necessary to add a error log here?
```
if (rc != 0) {
         rc = M0_ERR(rc);
         goto err;
}
```

Reply:
The rc from the computation will always be printed at line 139, I think it's enough.

Reply:
OK